Local build error 6.3.2

### Bug Description:

/data/manticoresearch/src/replication/wsrep_v25.cpp:359:8: error: declaration of ‘using Status_e = enum class RAW25::Status_e’ changes meaning of ‘Status_e’ [-fpermissive]
  359 |  using Status_e = Status_e;
      |        ^~~~~~~~
/data/manticoresearch/src/replication/wsrep_v25.cpp:18:12: note: ‘Status_e’ declared here as ‘enum class RAW25::Status_e’
   18 | enum class Status_e : int {
      |            ^~~~~~~~

/data/manticoresearch/src/replication/wsrep_v31.cpp:418:8: error: declaration of ‘using Status_e = enum class RAW31::Status_e’ changes meaning of ‘Status_e’ [-fpermissive]
  418 |  using Status_e = Status_e;
      |        ^~~~~~~~
/data/manticoresearch/src/replication/wsrep_v31.cpp:18:12: note: ‘Status_e’ declared here as ‘enum class RAW31::Status_e’
   18 | enum class Status_e : int {
      |            ^~~~~~~~



But 6.2.12 is compile success

### Manticore Search Version:

master

### Operating System Version:

ubuntu20.04

### Have you tried the latest development version?

None

### Internal Checklist:

<sup>To be completed by the assignee. Check off tasks that have been completed or are not applicable.</sup>
<details>

- [x] Implementation completed
- [x] Tests developed
- [x] Documentation updated
- [x] Documentation reviewed
- [x] Changelog updated

</details>

--- a/src/replication/wsrep_cxx_int.h
+++ b/src/replication/wsrep_cxx_int.h
@@ -98,7 +98,7 @@ extern std::atomic<uint64_t> uWritesetConnIds;
 template<typename WSREPWRAP>
 class Writeset_T final: public Wsrep::Writeset_i
 {
-	using Status_e = typename WSREPWRAP::Status_e;
+	using Status_e = typename WSREPWRAP::Status_e_;
 	CSphRefcountedPtr<WSREPWRAP> m_pWsrep;
 	uint64_t m_uConnId;
 	Wsrep::TrxMeta_t m_tMeta;
@@ -234,7 +234,7 @@ template<typename WSREPWRAP>
 class Provider_T: public Wsrep::Provider_i
 {
 protected:
-	using Status_e = typename WSREPWRAP::Status_e;
+	using Status_e = typename WSREPWRAP::Status_e_;
 	CSphRefcountedPtr<WSREPWRAP> m_pWsrep;
 	Wsrep::Cluster_i* m_pCluster;
 	Wsrep::ReceiverRefPtr_c m_pReceiver;

--- a/src/replication/wsrep_v25.cpp
+++ b/src/replication/wsrep_v25.cpp
@@ -356,7 +356,7 @@ struct WrappedWsrep_t final : public ISphRefcountedMT
 	}
 
 public:
-	using Status_e = Status_e;
+	using Status_e_ = Status_e;
 	inline static const char* szGetStatus ( Status_e eStatus ) noexcept
 	{
 		return GetStatus ( eStatus );

--- a/src/replication/wsrep_v31.cpp
+++ b/src/replication/wsrep_v31.cpp
@@ -415,7 +415,7 @@ struct WrappedWsrep_t final : public ISphRefcountedMT
 	}
 
 public:
-	using Status_e = Status_e;
+	using Status_e_ = Status_e;
 	inline static const char* szGetStatus ( Status_e eStatus ) noexcept
 	{
 		return GetStatus ( eStatus );

